xunit-viewer.js

```diff
@@ -14,7 +15,6 @@ export default async (args) => {
 
   const results = args.results
   if (!fs.existsSync(results)) {
-    const { showHelp } = import('./src/cli/args.js')
     showHelp()
```

Comment:
With this fix, I see the error below when I run the following command (the test_logs directory exists, but foo.xml does not)
`xunit-viewer -r test_logs/foo.xml`
```
 TypeError: Cannot write private member to an object whose class did not declare it
    at O (/Users/williamwallace/.config/yarn/global/node_modules/yargs/build/index.cjs:1:3385)
    at showHelp (/Users/williamwallace/.config/yarn/global/node_modules/yargs/build/index.cjs:1:42722)
    at default (file:///Users/williamwallace/.config/yarn/global/node_modules/xunit-viewer/xunit-viewer.js:18:5)
    at file:///Users/williamwallace/.config/yarn/global/node_modules/xunit-viewer/bin/xunit-viewer.js:6:1
    at ModuleJob.run (node:internal/modules/esm/module_job:222:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:323:24)
    at async loadESM (node:internal/process/esm_loader:28:7)
    at async handleMainPromise (node:internal/modules/run_main:120:12)
```